lib/client.js

```diff
@@ -11,15 +11,14 @@ function Client(opts) {
   self.opts = opts || {};
   self.baseUrl = defaultBaseUrl;
 
-  function useToken (token) {
+  function useToken (token, cb) {
     if (!token || typeof token !== 'string') {
```

Comment:
Actually, I didn't realize we were using `cb` to return an error if a key or token isn't passed in (there were some bugs reported on it awhile back). We can leave it like this for now and maybe clean it up later. Thanks!